libraries/standard/mqtt/utest/README.md

```diff
@@ -0,0 +1,118 @@
+# Running Unit Tests
+
+1. In your GNU compatible environment like WSL, mingw, Linux OS, MAC OS, etc., open a terminal.
```

Comment:
Looks like we will need to delete these examples of environments or else we have to test them all. So just "In your GNU compatible environment, open a terminal."